renku/core/management/repository.py

```diff
@@ -417,3 +419,42 @@ def init_repository(self, name=None, force=False):
             metadata.updated = datetime.now(timezone.utc)
 
         return str(path)
+
+    def init_empty_repository(self, force=False):
```

Comment:
Why not use/reuse `init_repository`? There is now some code duplication here.

Reply:
right, I changed the logic quite a bit, I'll replace the `init_repository` once this PR is ready